### DISCUSSION TITLE
delete the existinig hubs instance

### DISCUSSION THREAD

**Discussion by jsdude0220** [mozilla/hubs#3590] [Avatar Questions] [answered]
Hello,
I have one old hubs instance, but I can't use it because it was crashed.
so I would like to delete it and create the new hubs with old hubs domain.
what is the way to do this?
Please let me know who knows this.
Thanks in advance.

**Comment by robinkwilson** [comment]
Can you tell me more about how your Hubs Cloud instance is crashing? Do you have any console logs in the browser's developer tools?

If you'd still like to delete your Hubs Cloud and remake it with the old domain: 

**To delete your hub stack + resources**
1. Go to AWS Console > **CloudFormation** > **Stacks** (make sure you're in the right region)
2. Select your Hubs Cloud stack
3. Press Delete
4. Confirm

After that you've successfully deleted your Hub's Cloud and are ready to start another one on the same domain!

**Comment by jsdude0220** [comment]
hello, thanks for your response.
I have just deleted my old domain and create it again.
Even though it is not yet created, but I have one question, so Is this right to delete the domain and recreate?
I have created the domain again with same old domain.
So in this case, if I make hubs cloud again in this domain, it will be work well?
Of course, I have followed your description, so I have deleted my Hubs Cloud.
I would like to hear from you about this case.
Thanks in advance.

**Comment by robinkwilson** [comment, marked as answer]
Please don't delete your domain and recreate! You only have to worry about deleting the "hubs cloud stack" in cloudformation. :)

**Comment by jsdude0220** [comment]
That's great.
Thanks for your help!
